app/src/models/CityUser.ts

```diff
@@ -51,6 +51,7 @@ export class CityUser
           type: DataTypes.UUID,
           allowNull: false,
           primaryKey: true,
+          defaultValue: Sequelize.UUIDV4,
```

Comment:
We should do this for most models probably. Makes it easier to create new entries without having to manually create a `randomUUID` each time 👍